manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
The shebang line should not have a space between `python` and the path. It should be `#!/usr/bin/env python`.